web/vital_records/models.py

```diff
@@ -177,24 +177,24 @@ class VitalRecordsRequest(models.Model):
     status = FSMField(default="initialized", choices=STATUS_CHOICES)
     fire = models.CharField(max_length=50, choices=FIRE_CHOICES)
     relationship = models.CharField(max_length=50, choices=RELATIONSHIP_CHOICES)
-    legal_attestation = models.CharField(max_length=100)
-    first_name = models.CharField(max_length=100)
-    middle_name = models.CharField(max_length=100, blank=True)
-    last_name = models.CharField(max_length=100)
+    legal_attestation = models.CharField(max_length=257)
+    first_name = models.CharField(max_length=128)
+    middle_name = models.CharField(max_length=128, blank=True)
+    last_name = models.CharField(max_length=128)
     county_of_birth = models.CharField(max_length=15, choices=COUNTY_CHOICES)
     date_of_birth = models.DateField(null=True)
-    parent_1_first_name = models.CharField(max_length=100)
-    parent_1_last_name = models.CharField(max_length=100)
-    parent_2_first_name = models.CharField(max_length=100, blank=True)
-    parent_2_last_name = models.CharField(max_length=100, blank=True)
+    parent_1_first_name = models.CharField(max_length=128)
+    parent_1_last_name = models.CharField(max_length=128)
+    parent_2_first_name = models.CharField(max_length=128, blank=True)
+    parent_2_last_name = models.CharField(max_length=128, blank=True)
     number_of_records = models.IntegerField(choices=NUMBER_CHOICES, null=True)
-    order_first_name = models.CharField(max_length=50)
-    order_last_name = models.CharField(max_length=50)
+    order_first_name = models.CharField(max_length=128)
+    order_last_name = models.CharField(max_length=128)
```

Comment:
The #234 PR _should_ have had this change in it. But I don't know why/how it did not get into that PR! Putting it in this PR 😰

Reply:
Oops, and I didn't notice when I reviewed it :frowning: